aspnetcore-webapi/RequestBodyRead/Tests/UserControllerTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Text;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Newtonsoft.Json;
+using RequestBodyRead;
+using RequestBodyRead.Entities.Models;
+using RequestBodyRead.Repository;
+using RequestBodyRead.Shared.DataTransferObjects;
+using Xunit;
+
+namespace Tests
+{
+    public class UserControllerTests : IClassFixture<WebApplicationFactory<Program>>
+    {
+        private readonly WebApplicationFactory<Program> _factory;
+
+        public UserControllerTests(WebApplicationFactory<Program> factory)
+        {
+            _factory = factory;
+        }
+
+        [Fact]
+        public async void WhenGetAllUsers_ContainsThreeOriginalUsers()
+        {
+            var client = _factory.CreateClient();
+
+            client.DefaultRequestHeaders
+                .Accept
+                .Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+
+            var url = "/api/users/";
+            var response = await client.GetAsync(url);
+            var content = await response.Content.ReadAsStringAsync();
+            List<User>? users = JsonConvert.DeserializeObject<List<User>>(content);
+
+            var guids = users.Select(x => x.Id).ToList();
+            var guidsExpected = DataBaseMockup.Users.Select(x => x.Id).ToList();
+
+            foreach (var guid in guidsExpected)
+                Assert.True(guids.Contains(guid));
```

Comment:
Instead of `Assert.True`, we should use `Assert.Contains`. Moreover, we can skip the `foreach` loop by writing our assertion like so:

```csharp
Assert.Contains(guidsExpected, guids.Contains);
```